### DISCUSSION TITLE
problema endstop

### DISCUSSION THREAD

**Discussion by dnlucio** [community/community#46523] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

buon giorno ho un problema ad attivatre gli endstop e sensore filamento su marlin 2.0 ho una mks robin nano v3.1 endstop a 3 pin meccanico[tipo crality] e sensore filamento 3 pin. allego il configuration.h qualcuno puo aiutarmi a capire cosa sto sbagliando grazie

**Comment by queenofcorgis** [comment]
Hi @dnlucio ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Marlin. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
